exporters/jaeger/src/recordable.cc

```diff
@@ -117,9 +120,10 @@ void Recordable::SetResource(const opentelemetry::sdk::resource::Resource &resou
 {
   // only service.name attribute is supported by specs as of now.
   auto attributes = resource.GetAttributes();
-  if (attributes.find("service.name") != attributes.end())
+  if (attributes.find(OTEL_CPP_GET_ATTR(AttrServiceName)) != attributes.end())
   {
-    service_name_ = nostd::get<std::string>(attributes["service.name"]);
+    service_name_ =
+        nostd::get<std::string>(attributes[OTEL_CPP_GET_ATTR(AttrServiceName()]);
```

Comment:
Is parenthesis needed in `AttrServiceName()`?

Reply:
No, it was a typo. I should start testing Jaeger too locally now :). Fixed now.